docs/reference/data-streams/logs.asciidoc

```diff
@@ -83,23 +84,19 @@ The `min` mode sorts indices by the minimum value of multi-value fields.
 Missing values are sorted to appear `_first`.
 
 You can override these default sort settings. For example, to sort on different fields
-and change the order, modify `index.sort.field` and `index.sort.order`.
+and change the order, manually configure `index.sort.field` and `index.sort.order`. For more details, see
+<<index-modules-index-sorting>>.
 
 When using the default sort settings, the `host.name` field is automatically injected into the index mappings as a `keyword` field to ensure that sorting can be applied. This guarantees that logs are efficiently sorted and
 retrieved based on the `host.name` and `@timestamp` fields.
 
-NOTE: If `subobjects` is set to `true` (default), the `host.name` field is mapped as an object field
+NOTE: If `subobjects` is set to `true` (default), the `host` field is mapped as an object field
 named `host` with a `name` child field of type `keyword`. If `subobjects` is set to `false`,
 a single `host.name` field is mapped as a `keyword` field.
 
-After an index is created, the sort settings cannot be modified. To use different sort settings,
-create a new index. For data streams, you can update the relevant component templates and then 
+To apply different sort settings, update the data stream's component templates, and then 
```

Comment:
could be a followup - I see you removed the index context from this sentence. does that mean that certain index modes work with _only_ data streams? if so, we should add that info to the list of modes.